satellite/satellitedb/projectmembers.go

```diff
@@ -48,8 +48,8 @@ func (pm *projectMembers) GetByProjectID(ctx context.Context, projectID uuid.UUI
 				INNER JOIN users u ON pm.member_id = u.id
 					WHERE pm.project_id = ?
 					AND ( u.email LIKE ? OR 
-						  u.first_name || u.last_name LIKE ? OR
-						  u.last_name || u.first_name LIKE ? )
+						  u.full_name LIKE ? OR
```

Comment:
Assume you have 
u.full_name = 'Bohdan Artemenko', 
search_query = 'artemenko'
Will this code return 'Bohdan Artemenko' user?

Reply:
@Barterio no, we should change this query, we will create a bug in our backlog